src/pages/Home.jsx

```diff
@@ -1,5 +1,89 @@
+import { Link } from "react-router-dom";
+
 function Home() {
-  return <h1>Home</h1>;
+  return (
+    <main className="center-hv">
+      <div className="container">
+        <div>
+          <h1 className="main-title">
+            <span>
+              <img
+                className="logo"
+                src="https://img.icons8.com/ios/50/000000/retro-tv.png"
+                alt="logo"
+              />
+            </span>
+            /<span className="title-empha">Stre</span>Flix
+          </h1>
+          <p className="main-slogan">{"//"}Stream at peace</p>
+          <p>
+            Unlimited Movies, TV shows and more. Stream videos at peace, at your
+            convenience. Drag your couch and start watching!!
+          </p>
+
+          <Link to="/explore">
+            <button className="btn btn-primary">Explore Videos</button>
+          </Link>
+          <button className="btn btn-secondary ml-2">
+            <a href="#category">Stream by Categories</a>
+          </button>
+        </div>
+        <img
+          className="landing-img"
+          src="https://assets.website-files.com/5d5e2ff58f10c53dcffd8683/5d5e304c8f10c501d7fd8a1d_meditating.svg"
```

Comment:
this is very cool , love the overall UI. do check contrast value to your primary color to white, only nitpicking doubt I found.